src/game.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable max-len */
+/* eslint-disable no-console */
```

Comment:
You should remove the eslint-disable for max-len. This rule is there to enforce readability and maintainability. Consider refactoring your code to follow this rule.